docs/docs-content/clusters/pcg/manage-pcg/configure-proxy.md

```diff
@@ -0,0 +1,208 @@
+
+## Prerequisites
+
+- A PCG is deployed into an active Kubernetes cluster, which is active and healthy. Refer to
+  [Deploy a PCG to an Existing Kubernetes Cluster](../deploy-pcg-k8s.md) for additional guidance.
+
+  :::warning
+
+  If you deployed a [PCG through the Palette CLI](../pcg.md#supported-environments), refer to the respective platform
+  install guide for instructions on how to configure proxy settings during the installation process through the CLI.
+
+  :::
+
+- The kubeconfig file for the Kubernetes cluster where the PCG is deployed. The Kubconfig file is used to authenticate
+  with the Kubernetes cluster and deploy the Reach service.
+
+- Admin access to the Kubernetes cluster where the PCG is deployed. The Reach Helm Chart will create a namespace,
+  service accounts, and roles in the cluster.
+
+- The extract utilities `zip` and `tar` are installed on the machine you are using to deploy the Helm chart.
+
+- Tenant administrator access.
+
+- Proxy configuration details, such as the proxy URL, port, and authentication credentials.
+
+- The deployed PCG must have network connectivity to the proxy server.
+
+- The Kubernetes cluster where the PCG is deployed must have the proxy configuration. This includes any Certificate
+  Authority (CA) certificates are required to authenticate the proxy server. This step varies depending on the platform
+  where the Kubernetes cluster is deployed. Some platforms, such as managed Kubernetes services, may require additional
+  steps. Below is a list of helpful links to set up proxy configurations for some common Kubernetes platforms:
+
+  - [Amazon EKS](https://repost.aws/knowledge-center/eks-http-proxy-containerd-automation)
+  - [Azure AKS](https://learn.microsoft.com/en-us/azure/aks/http-proxy)
+  - [vSphere](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-configuration/GUID-7576FBBE-8B67-4D3F-B859-C9688E9442D7.html)
+
+    :::warning
+
+    This feature has only been tested on AWS, Azure, and VMware vSphere. If you are using a different cloud provider,
+    contact our support team for additional guidance.
+
+    :::
+
+- Download the Reach Helm Chart provided by our support team. The Reach Helm Chart is used to deploy the Reach service
+  into the Kubernetes cluster where the PCG is deployed. The Reach service is used to manage proxy configurations for
+  the PCG. Contact our support team to obtain the Reach Helm Chart.
+
+- `cert-manager` is not already deployed in the Kubernetes cluster where the PCG is deployed. If `cert-manager` is
+  already deployed, make sure you edit the deployment to use the
+  `- --feature-gates=AdditionalCertificateOutputFormats=true` flag. Otherwise, the Reach service cannot manage the proxy
+  configurations.
+
+## Enable Proxy
+
+1. Open a terminal session and navigate to the folder where you downloaded the Reach Helm Chart zip file.
+
+2. Unzip the downloaded artifact you received from the support team.
```

Comment:
Are you referring to the Reach helm zip? I find the word "artifact" to be confusing here, since we're talking about a specific file in the previous step and call out an ambiguous artifact in the next 🤔